src/main/resources/application-test.properties

```diff
@@ -1,2 +1,5 @@
 server.port=8090
-agents.url=http://localhost:8080/user
\ No newline at end of file
+agents.url=http://localhost:8080/user
+
+kafka.topic=INCIDENCES
+spring.kafka.producer.bootstrap-servers=localhost:9092
```

Comment:
Just a doubt, kafka default port is `2181` why change it to 9092?

Reply:
@ByBordex can you explain if its compulsory to use 9092? just for the ci to work properly.

Reply:
I've just followed the startup guide about [sending messages](https://kafka.apache.org/quickstart#quickstart_send) which uses such port.

Reply:
okey, i guess it works haha

Reply:
@thewilly I've made some research and it seems that 2181 is the port of zookeper. So kafka relies in zookeper port but listens to 9092 by defaut.